Loop TodayExtension/TodayExtensionContext.swift

```diff
@@ -42,113 +42,115 @@ struct GlucoseContext {
     var sensor: SensorDisplayable?
 }
 
-class TodayExtensionContext {
-    var data: [String:Any] = [:]
-    let storage = UserDefaults(suiteName: "group.com.loudnate.Loop")
+final class TodayExtensionContext: NSObject, RawRepresentable {
+    typealias RawValue = [String: Any]
+    private let version = 1
     
-    var glucose: GlucoseContext? {
-        get {
-            if data["gcgv"] == nil { return nil }
-            
-            var sensor: SensorDisplayableContext? = nil
-            if data["gcsv"] != nil {
-                sensor = SensorDisplayableContext(
-                    isStateValid: data["gcsv"] as! Bool,
-                    stateDescription: data["gcsd"] as! String,
-                    trendType: data["gcst"] as? GlucoseTrend,
-                    isLocal: data["gcsl"] as! Bool)
-            }
-            return GlucoseContext(
-                latest: HKQuantitySample(
-                    type: HKQuantityType.quantityType(forIdentifier: HKQuantityTypeIdentifier.bloodGlucose)!,
-                    quantity: HKQuantity(unit: HKUnit.milligramsPerDeciliterUnit(),
-                                         doubleValue: data["gcgv"] as! Double),
-                    start: data["gcgd"] as! Date,
-                    end: data["gcgd"] as! Date),
-                sensor: sensor)
-        }
-        set(lgv) {
-            data["gcgv"] = lgv?.latest.quantity.doubleValue(for: HKUnit.milligramsPerDeciliterUnit())
-            data["gcgd"] = lgv?.latest.startDate
-            data["gcsv"] = lgv?.sensor?.isStateValid
-            data["gcsd"] = lgv?.sensor?.stateDescription
-            data["gcst"] = lgv?.sensor?.trendType
-            data["gcsl"] = lgv?.sensor?.isLocal
-        }
+    var latestGlucose: GlucoseContext?
```

Comment:
I tried that and get the following errors:

1. `StatusExtensionContext.swift: Property 'self.latestGlucose' not initialized at super.init call`
2. `StatusExtensionDataManager.swift: Cannot assign to property: 'latestGlucose' is a 'let' constant`

ideas?